mod/wikirate_source/set/type/source/source_preview.rb

```diff
@@ -54,7 +54,7 @@
     elsif file_card
       if mime_type = file_card.file.content_type and ( mime_type == "application/pdf" or mime_type.start_with?("image/") )
         if mime_type == "application/pdf"
-          content_tag(:div, '<iframe id="source-preview-iframe" src="pdfjs/viewer.html?file='+ file_card.attachment.url+'"  security="restricted" sandbox="allow-same-origin allow-scripts allow-forms" ></iframe>', {:id=>"pdf-preview", :class=> "webpage-preview"},false)
+          content_tag(:div, '<iframe id="source-preview-iframe" src="files/viewer.html?file='+ file_card.attachment.url+'"  security="restricted" sandbox="allow-same-origin allow-scripts allow-forms" ></iframe>', {:id=>"pdf-preview", :class=> "webpage-preview"},false)
```

Comment:
Line is too long. [268/80]<br>Use the new Ruby 1.9 hash syntax.<br>Space missing after comma.<br>Surrounding space missing for operator '+'.<br>Surrounding space missing for operator '=>'.<br>Space inside { missing.<br>Space inside } missing.